packages/kiota_abstractions/lib/src/serialization/untyped_array.dart

```diff
@@ -0,0 +1,15 @@
+part of '../../kiota_abstractions.dart';
+
+/// Represents an untyped node with a collection of other untyped nodes.
+class UntypedArray extends UntypedNode {
+
+  /// Constructs an instance from the supplied [collection]
+  UntypedArray(this.collection);
+
+  Iterable<UntypedNode> collection;
```

Comment:
Any reason why not `final`? Could mark the constructor `const`

Reply:
I see most other constructors could be `const` as well

Reply:
Made all untyped constructors const